client/signup/steps/user/index.jsx

```diff
@@ -401,6 +390,10 @@ export class UserStep extends Component {
 			} );
 		}
 
+		if ( this.isSimplerMobileForm() && this.isEmailForm() ) {
+			return translate( 'Create Account' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Create account.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/user-registration-mobile-social-links)